src/solver/conv_winoRxS_f2x3.cpp

```diff
@@ -572,15 +578,25 @@ ConvBinWinogradRxSf2x3::GetSolution(const ConvolutionContext& params,
     };
     kernel.comp_options = options.GenerateFor(kbp::GcnAsm{});
 
-    std::string kernel_name    = "miopenSp3AsmConv";
-    std::string kernel_file    = "Conv_Winograd";
-    std::string kernel_postfix = "_v21_1_2_gfx9";
+    std::string kernel_name = "miopenSp3AsmConv_v21_1_2";
+    std::string kernel_file = "Conv_Winograd_v21_1_2";
+    std::string kernel_postfix;
+
+    if(is_gfx9)
+    {
+        kernel_name += "_gfx9";
+    }
+    else // if(StartsWith(name, "gfx10"))
+    {
+        kernel_name += "_gfx10";
+        kernel.comp_options += std::string(" -mcumode -mwavefrontsize64");
+    }
 
     if(params.IsFp32())
-        kernel_postfix += "_fp32";
+        kernel_postfix = "_fp32";
     else
     {
-        kernel_postfix += "_fp16_dot2_edc";
+        kernel_postfix = "_fp16_dot2_edc";
```

Comment:
Let's be as expressive as C/C ++ allows us.
```suggestion
        const std::string kernel_postfix = params.IsFp32() ? "_fp32" : "_fp16_dot2_edc";
```

Reply:
Thank you for your help, fixed in 148bb84